recipes/periodicgw/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "periodicgw" %}
+{% set version = "1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/periodicgw-{{ version }}.tar.gz
+  sha256: 0c83bf0d2c5a7b082ad543d4225f89f97e00b1a717b2b2247f04345f5ea57be7
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - numpy
+    - python >=3.6
+    - scipy
+
+test:
+  imports:
+    - periodicgw
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/scottkalevhansen/periodic-gw
```

Comment:
This URL gives me a 404, is it correct?

Reply:
The URL gives a private repository by far. We will make it public soon.

Reply:
I just changed the visibility to public.